### ISSUE TITLE
1.10 Material - Cards styling

### ISSUE THREAD

**Issue by bosschaertB** [usebedrock/bedrock#251]
Is there a reason the card component has no fixed width?

**Comment by Wolfr** [comment]
I think they want to provide a base that is as reusable as possible.

I guess in our examples we can use it in a grid context:

````
<div class="mdc-layout-grid">
    <div class="mdc-layout-grid__inner">
        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6">
            <!-- Your card -->
       </div>
    </div>
</div>
````

So - idea - perhaps we can make a "movie cards" example, where we put a few media cards in there?

**Comment by Wolfr** [comment]
I put the layout grid around the example.

**Comment by bosschaertB** [comment]
Fix icon buttons 7535fa15a8038f8d67d37c92ed3d462a1469f8c8
